templates/boltzmann.js.tmpl

```diff
@@ -1,15 +1,17 @@
-/* <% if not test %>eslint-disable<% endif %> */
-/* <% if not test %>istanbul ignore file<% endif %> */
+#!/usr/bin/env node
+{% if selftest == false %}/* eslint-disable */{% endif %}
+{% if selftest == false %}/* istanbul ignore file */{% endif %}
+
 'use strict'
 
-// <% if honeycomb %>
+// {% if honeycomb == "on" %}
```

Comment:
(I'm starting to think we should process the flags a bit more before sending them in as context.)